Makefile

```diff
@@ -46,17 +50,52 @@ test: ## Tests a previous build docker image to see if starts
 	@echo "[test] Cleaning up test container $(NAME_TEST_CONTAINER)"
 	@docker rm -f $(NAME_TEST_CONTAINER) > /dev/null 2>&1 || true
 
-tag: ## Tags a local build image to make it ready for push to docker registry
-	docker tag -f $(shell docker images -q $(NAME_BUILD_CONTAINER)) $(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)$(DOCKER_REGISTRY_IMAGE_TAG_VERSION)
+tag: ## Tags local build image to make it ready for push to docker registry
+	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}${DOCKER_REGISTRY_IMAGE_VERSION}"
+	docker tag "$(shell docker images -q ${NAME_BUILD_CONTAINER})" "${DOCKER_REGISTRY_PREFIX}:${DOCKER_REGISTRY_IMAGE_TAG_PREFIX}latest"
 
+push: ## Push local versioned and latest images to the docker registry
+	docker push "$(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)$(DOCKER_REGISTRY_IMAGE_VERSION)"
+	docker push "$(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)latest"
 
-push: ## Push the local image to the docker registry
-	docker push $(DOCKER_REGISTRY_PREFIX):$(DOCKER_REGISTRY_IMAGE_TAG_PREFIX)$(DOCKER_REGISTRY_IMAGE_TAG_VERSION)
+publish: ## Publish npm package
+	npm publish
+
+bump-major: ## Bump the major version (1.0.0 -> 2.0.0)
+	@echo "Current version: ${DOCKER_REGISTRY_IMAGE_VERSION}"
+	@DOCKER_REGISTRY_IMAGE_VERSION="$(shell docker run --rm -v "${PWD}":/app treeder/bump major)"
+	@#	Don't add git tag and commit
+	@npm version --no-git-tag-version major
+	@echo "New version: ${DOCKER_REGISTRY_IMAGE_VERSION}"
+	@git add VERSION package.json
+	@git commit -m "Bump version to ${DOCKER_REGISTRY_IMAGE_VERSION}"
+
+bump-minor: ## Bump the minor version (0.1.0 -> 0.2.0)
+	@echo "Current version: ${DOCKER_REGISTRY_IMAGE_VERSION}"
+	@DOCKER_REGISTRY_IMAGE_VERSION="$(shell docker run --rm -v "${PWD}":/app treeder/bump minor)"
+	@#	Don't add git tag and commit
+	@npm version --no-git-tag-version minor
+	@echo "New version: ${DOCKER_REGISTRY_IMAGE_VERSION}"
+	@git add VERSION package.json
+	@git commit -m "Bump version to ${DOCKER_REGISTRY_IMAGE_VERSION}"
+
+bump-patch: ## Bump the patch version (0.0.1 -> 0.0.2)
+	@echo "Current version: ${DOCKER_REGISTRY_IMAGE_VERSION}"
```

Comment:
https://stackoverflow.com/a/3477400/521361

So is `@echo` an echo that won't echo anything?

Reply:
It certainly looks paradoxical 😄 
`@` suppresses the printing out of the **command**, without it the output of that command will be
```
echo "Current version: 0.9.0"
Current version: 0.9.0
```
with it
```
Current version: 0.9.0
```